src/controllers/article.js

```diff
@@ -378,6 +376,7 @@ export default {
   deleteArticle: async (request, response) => {
     const { slug } = request.params;
     await Articles.deleteArticle(slug);
+
     return response.status(200).json({ status: 'success', data: {}, message: 'Article deleted succesfully' });
```

Comment:
data should be undefined i.e
```suggestion
    return response.status(200).json({ status: 'success', message: 'Article deleted succesfully' });
```